holidayextras/brands
# Nb 654

#### What does this PR do? (please provide any background)
Revised legoland mappings.scss file to move relevant properties into brands.

#### How can this be tested?
- Checkout this branch and npm link [NB-654](https://github.com/holidayextras/brands/pull/480)
- Legoland should look exactly the same as production.

#### What tests does this PR have?
None 

#### How can this be tested?
- Checkout this branch and npm link [NB-654](https://github.com/holidayextras/the-wall/pull/604)
- Legoland should look exactly the same as production.

#### Any tech debt?

#### Screenshots / Screencast

#### What gif best describes how you feel about this work?
![]()


---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.
